src/components/Nav/Nav.js

```diff
@@ -1,7 +1,57 @@
 import React, { Component } from 'react';
+import NavMain from './NavMain/NavMain';
+import NavCategory from './NavCategory/NavCategory';
+import SerchModal from './SerchModal/SerchModal';
 
 export default class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isLocationButton: false,
+      isSerch: false,
+      categotyList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('data/navListData.json')
+      .then(res => res.json())
+      .then(result =>
+        this.setState({
+          categotyList: result,
+        })
+      );
+  }
+
+  handleLocationButton = () => {
+    const { isLocationButton } = this.state;
+    this.setState({
+      isLocationButton: !isLocationButton,
+      isSerch: false,
+    });
+  };
+
+  handleSerchButton = () => {
+    const { isSerch } = this.state;
+    this.setState({
+      isSerch: !isSerch,
+      isLocationButton: false,
+    });
+  };
+
   render() {
-    return <div>Nav입니다</div>;
+    const { isLocationButton, categotyList } = this.state;
+    return (
+      <>
+        <NavMain
+          handleLocationButton={this.handleLocationButton}
+          handleSerchButton={this.handleSerchButton}
+        />
+        {isLocationButton === true ? (
+          <NavCategory categotyList={categotyList} />
+        ) : undefined}
```

Comment:
```suggestion
        {isLocationButton && 
          <NavCategory categotyList={categotyList} />
        }
```
이렇게 표현해도 됩니다 왜 위와 같이 표현해도 되는지 코멘트로 답글 한번 달아주세요!

Reply:
`&&`는 논리 연산자로 앞에 조건이 `true`일때 출력이 되고 `false`일때는 react에서 출력을 안하도록 되어있어서 사용하는거 같습니다!